core/src/test/java/org/apache/iceberg/actions/TestBinPackStrategy.java

```diff
@@ -60,7 +60,7 @@ public Table table() {
     }
 
     @Override
-    public Set<DataFile> rewriteFiles(String groupId, List<FileScanTask> filesToRewrite) {
```

Comment:
We are pulling this because we removed "groupID" state and put it into the strategy implementations